tool/grind.dart

```diff
@@ -67,7 +67,7 @@ Future<FlutterRepo> get cleanFlutterRepo async {
     await cleanFlutterDir.parent.create(recursive: true);
     assert(_lockFuture == null);
     _lockFuture = new File(pathLib.join(cleanFlutterDir.parent.path, 'lock'))
-        .openSync(mode: FileMode.WRITE)
+        .openSync(mode: FileMode.write)
```

Comment:
This will mean that dartdoc can only run on the very very latest sdk; you might want to just live with deprecation warnings for a while.

Reply:
There are enough other times I've done this that it probably is already required; updated minimums on pubspec.yaml.